packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -40,7 +40,14 @@ const { Logger, delay } = require("@uma/financial-templates-lib");
 let logger;
 let spokeUrl;
 let customNodeUrl;
-let hubConfig = { configRetrieval: "localStorage", saveQueriedBlock: "localStorage", spokeRunner: "localStorage" };
+let hubConfig = {};
+
+const defaultHubConfig = {
```

Comment:
this small change was made to make it cleaner when overriding configs, especially between test itterations.